### PR TITLE
[TSK-89] implement cards crud

### PR DESCRIPTION
**Pull request by kostya4201337**

### CODE REVIEW

File: client/src/pages/workspace/board/addCard/AddCard.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        validateCard(card);
+    }, [card, boardId]);
+
+    const validateCard = (card: ICardCreation) => {
+        if (boardId >= 0 && card.name !== "" && card.description !== "") {
+            setIsValid(true)
+        } else {
+            setIsValid(false)
+        }
+    }
```
Comment (Hoocki) [inline client/src/pages/workspace/board/addCard/AddCard.tsx]: why are you checking `boardId >= 0` if boardId never be less then 0? When you delete or add board your boardId also, can't be less then 0

File: client/src/services/card.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+export const getCardsService = (boardId: number): ICard[] => {
+    return [
+        {id: 1, boardId: 1, name: 'card1', description: 'desc1', createdAt: new Date(), updatedAt: new Date()},
+        {id: 2, boardId: 1, name: 'card2', description: 'desc2', createdAt: new Date(), updatedAt: new Date()},
+        {id: 3, boardId: 2, name: 'card3', description: 'desc3', createdAt: new Date(), updatedAt: new Date()}
+    ];
+};
+
+export const addCardService = (boardId: number, newBoard: ICardCreation): ICard => {
```
Comment (Hoocki) [inline client/src/services/card.ts]:
```suggestion

export const addCardService = (boardId: number, newCard: ICardCreation): ICard => {
```

**Review by adsukharev — APPROVED** [review]
I approved but did not check the PR. Roma will be 100% repsonsible for the review

File: client/src/pages/workspace/board/card/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                </IconButton>
+            </div>
+            <MutableInput className={"description"} text={description} placeHolder="" setText={setDescription}/>
+            <div className={"inline cardBottomInfo"}>
+                <div className={"taskId"}>TSK-{card.id}</div>
+                <div className={"date"}>
+                    {('0' + card.createdAt.getDate()).slice(-2)}.
+                    {('0' + (card.createdAt.getMonth() + 1)).slice(-2)}.
+                    {card.createdAt.getFullYear()}
+                </div>
```
Comment (Hoocki) [inline client/src/pages/workspace/board/card/Card.tsx]: it may be better to make a date format change in a separate method

**Review by Hoocki — COMMENTED** [review]
image
as you can see, when the number of cards in the board is greater than the size of the screen, scrolling to the whole page is triggered. Do a scroll inside the board

**Review by Hoocki — APPROVED** [review]

**Pull request merged**